db/seeds.rb

```diff
@@ -30,6 +34,7 @@
   ])
 
 Address.create!([
+  {line_one: '1510 Blake Street', line_two: nil, city: 'Denver', state: 'CO', zip: '80202', country: 'USA', user_id: User.first.id},
```

Comment:
Line is too long. [132/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.